docs/FunctionalTestBuildScripts.md

```diff
@@ -0,0 +1,13 @@
+# Build Scripts
+
+This file records scripts used to compile the test files, by alphabetical order.
```

Comment:
>files [](http://example.com/codeflow?start=51&length=5)

Might be helpful to specify what types of test files this generates (ie. binaries + debug files) and/or add a sentence explaining these can be found in the functional test projects in BaselineTestsData and/or FunctionalTestsData.

Reply:
fix by adding a sentence explaining these can be found in what locations.